proxy/manager.go

```diff
@@ -3,14 +3,23 @@ package proxy
 import (
 	"context"
 	"net/http"
+	"strings"
 
 	"github.com/ONSdigital/dp-legacy-cache-proxy/config"
 	"github.com/ONSdigital/dp-legacy-cache-proxy/response"
 	"github.com/ONSdigital/log.go/v2/log"
 )
 
 func (proxy *Proxy) manage(ctx context.Context, w http.ResponseWriter, req *http.Request, cfg *config.Config) {
-	targetURL := cfg.BabbageURL + req.URL.String()
+	var targetURL string
```

Comment:
All this is a good candidate to be in its own function, similar to what you've done with `IsReleaseCalendarURL`, so that you can simply call like this (just an example, feel free to chose a different function name):
```golang
targetURL := getTargetURL(req.URL.String(), cfg)
```